utils/text_formatting.jsx

```diff
@@ -66,6 +67,9 @@ export function formatText(text, inputOptions) {
     } else if (!('markdown' in options) || options.markdown) {
         // the markdown renderer will call doFormatText as necessary
         output = Markdown.format(output, options);
+    } else if (options.linkOnlyMarkdown) {
```

Comment:
Instead of adding a separate `linkOnlyMarkdown` option that works similar to `removeMarkdown`, what if you added a `renderer` option instead that could be used for both? Then, whatever calls this could specify any number of different renderers

Reply:
Thanks for the review! Indeed your approach looks much better than mine. I will try to implement it. I have two questions. 

You wrote: 
> Then, whatever calls this could specify any number of different renderers without needing a new option added each time
Do you mean the new option `render` should be a list? Is anywhere a need to format a specific text by multiple renderers?

Function `sanitizeHtml` is called after a specific text is formatted by the `RemoveMarkdown` renderer. This isn't called after format by the `LinkOnlyRenderer`. Maybe we should create a new option to specify whether this function has to be called after formatting?

Reply:
> Do you mean the new option render should be an array? Is anywhere a need to format a specific text by multiple renderers?

No, it would only need to use a single renderer. I just meant that I could have one component use `<Markdown options={{renderer: new RemoveMarkdown()}}/>` and then another that uses `<Markdown options={{renderer: new LinkOnlyRenderer()}}/>`

> Maybe we should create a new option to specify whether this function has to be called after formatting?

I don't think we actually need to call `sanitizeHtml` after calling `formatWithRenderer(output, removeMarkdown)`. `marked` itself already sanitizes the output, so we shouldn't need that when using `RemoveMarkdown` either

Reply:
Thanks for the clarification! I will update the PR soon.